charts/osm/values.yaml

```diff
@@ -246,6 +310,20 @@ osm:
     # -- Image used for tracing
     image: jaegertracing/all-in-one
 
+    ## Node labels for pod assignment
+    ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+    nodeSelector:
+      kubernetes.io/arch: amd64
+      kubernetes.io/os: linux
+
+    ## Affinity settings for pod assignment
+    ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+    affinity: {}
```

Comment:
Let's do the same as Prometheus here and define all of the architectures supported by the image here in `nodeAffinity`.